src/Database/Dialect/MysqlDialectTrait.php

```diff
@@ -16,8 +16,7 @@
  */
 namespace Cake\Database\Dialect;
 
-use Cake\Database\Schema\BaseSchema;
-use Cake\Database\Schema\MysqlSchema;
+use Cake\Database\Schema\SchemaDialect;
```

Comment:
Why not have `SchemaDialect` class under `Database\Dialect` too?

Reply:
I thought the cake pattern was to put the base class in the parent directory.

Reply:
Well it's not in parent directory is it? :slightly_smiling_face: `Dialect` and `Schema` are sibling directories.

Reply:
It was before moving to Dialect is why I left it there. Easy fix.